fiftyone/core/utils.py

```diff
@@ -1285,6 +1285,10 @@ def deserialize_numpy_array(numpy_bytes, ascii=False):
     Returns:
         the numpy array
     """
+
+    if isinstance(numpy_bytes, list):
```

Comment:
When I originally upgraded the regression tests were failing (expected `bytes` and not `list`). It seems like the upgraded de-serialized the byes into a list. 

Making this  allowed the regressions to complete without further issue.